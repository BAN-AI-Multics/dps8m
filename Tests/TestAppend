de dsbr.u 1
de dsbr.addr 7000
de ppr.ic 0
de ppr.prr 0
de ppr.psr 0
de ppr.p 1
dpsinit segment table
dpsinit SDWAM
load Play/Testappend0.oct segment 0 address  0
load Play/Testappend1.oct segment 1 address  1000
load Play/Testappend2.oct segment 2 address  2000
load Play/Testappend3.oct segment 3 address  3000
;set debug stderr
;set cpu debug=trace;appending;regdumpaqi;regdumpppr;addrmod
;set cpu debug=trace;appending;regdumpaqi;regdumpppr;addrmod
dpsdump segment table
g 0
q
