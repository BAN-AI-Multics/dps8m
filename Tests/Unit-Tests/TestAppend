de dsbr.u 1
de dsbr.addr 1000
de ppr.ic 0
de ppr.prr 0
de ppr.psr 0
de ppr.p 1
dpsinit segment table
dpsinit SDWAM
load ./Testappend0.oct segment 0 address  0
load ./Testappend1.oct segment 1 address  0100
load ./Testappend2.oct segment 2 address  0200
load ./Testappend3.oct segment 3 address  0300
;set debug stderr
;set cpu debug=trace;appending;regdumpaqi;regdumpppr;addrmod
dpsdump segment table
g 0
q

