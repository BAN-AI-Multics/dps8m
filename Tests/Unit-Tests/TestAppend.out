
dps-8/m simulator V4.0-0 Beta        git commit id: f386776a
zero-initialized segments 0 .. 7
zero-initialized SDWAM
Writing SDW to address 00001000 (DSBR.ADDR+2*0 offset) 
61 (000075) words loaded into segment 0(0) at address 000000
Writing SDW to address 00001002 (DSBR.ADDR+2*1 offset) 
21 (000025) words loaded into segment 1(1) at address 000100
Writing SDW to address 00001004 (DSBR.ADDR+2*2 offset) 
26 (000032) words loaded into segment 2(2) at address 000200
Writing SDW to address 00001006 (DSBR.ADDR+2*3 offset) 
23 (000027) words loaded into segment 3(3) at address 000300
*** Descriptor Segment Base Register (DSBR) ***
DSBR: ADDR=001000 BND=00000 U=1 STACK=0000
*** Descriptor Segment Table ***
Seg 0 - ADDR=000000 R1=0 R2=0 R3=0 F=1 FC=2 BOUND=4 R=1 E=1 W=1 P=1 U=1 G=1 C=1 EB=100
Seg 1 - ADDR=000100 R1=0 R2=0 R3=0 F=1 FC=2 BOUND=2 R=1 E=1 W=1 P=1 U=1 G=1 C=1 EB=25
Seg 2 - ADDR=000200 R1=0 R2=0 R3=0 F=1 FC=2 BOUND=2 R=1 E=1 W=1 P=1 U=1 G=1 C=1 EB=32
Seg 3 - ADDR=000300 R1=0 R2=0 R3=0 F=1 FC=2 BOUND=4 R=1 E=1 W=1 P=1 U=1 G=1 C=1 EB=76
Seg 4 - *** Uninitialized ***
Seg 5 - *** Uninitialized ***
Seg 6 - *** Uninitialized ***
Seg 7 - *** Uninitialized ***
Hello world from TestAppend0 ....This is from Segment 1 ...From Segment 2 This is ...>Back at segment 0 we are!000000 000616000000 000123000000 000456000000 000003
cpuCycles = 364
TestAppend-16> g 0

DIS instruction, IC: 000010 (000036235052)
Goodbye
